src/main/webapp/components/BaseFormView.jsx

```diff
@@ -423,7 +514,11 @@ class BaseFormView extends PureComponent {
             >
                 {this.generateWarningMessage()}
                 {this.generateErrorMessage()}
+                {this.renderGroupElements()}
                 {this.entities.map((e) => {
+                    if (e.isGrouping) {
```

Comment:
nit: Please add inline comment here mentioning why we need to handle this condition.

Reply:
Done